examples/Connectivity.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Test the connectivity between all processes
+ * WIP
+ */
+
+import mpi.*;
+
+class Connectivity {
+    public static void main(String args[]) throws MPIException {
+        MPI.Init(args);
+
+        /*
+         * MPI_COMM_WORLD is the communicator provided when MPI is
+         * initialized. It contains all the processes that are created
+         * upon program execution.
+         */
+        int myRank = MPI.COMM_WORLD.getRank();
+        int numProcesses = MPI.COMM_WORLD.getSize();
+        Status status;
+        int verbose = 0;
+        int peerProcess;
+        String processorName = MPI.getProcessorName();
+
+        for (int i = 0; i < numProcesses; i++) {
+            if (myRank == i) { /* Find current process */
```

Comment:
I don't know the Java ecosystem at all -- is it more common for single-line comments to be `//` instead of `/* ... */`?